beetmoverscript/test/test_task.py

```diff
@@ -343,3 +350,24 @@ def test_get_release_props(context, mocker, taskjson, locale, relprops, expected
 def test_is_custom_beetmover_task(context, kind, expected):
     context.task['tags']['kind'] = kind
     assert is_custom_checksums_task(context) == expected
+
+
+@pytest.mark.parametrize("path, expected", ((
+    '/src/beetmoverscript/test/test_work_dir/cot/eSzfNqMZT_mSiQQXu8hyqg/public/build/target.mozinfo.json', 'eSzfNqMZT_mSiQQXu8hyqg'
+), (
+    'test_work_dir/cot/eSzfNqMZT_mSiQQXu8hyqg/public/build/target.mozinfo.json', 'eSzfNqMZT_mSiQQXu8hyqg'
+),))
+def test_get_taskId_from_full_path(path, expected):
```

Comment:
@srfraser  Can we please add a test for when the string `cot` lies within the task-id itself? Like
`/src/beetmoverscript/test/test_work_dir/cot/eSzfNqMZT_mSicotu8hyqg/public/build/target.mozinfo.json` should return `eSzfNqMZT_mSicotu8hyqg`

In the function definition, you're splitting based on the os separators, which makes sense. `cot` will be among the indexed items in that separated list - https://github.com/mozilla-releng/beetmoverscript/pull/185/files#diff-21f3f939e3375901322bd3639f4394bcR223 But for the safety, can we please add a test that makes sure the presence of the `cot` substring is not disruptive by any form to this logic?